### PR TITLE
volunteers form

### PR DESCRIPTION
**Pull request by azadi11**
I have created a form to register new volunteers

### CODE REVIEW

File: client/src/components/VolunteersForm.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+
+class VolunteersForm extends React.Component {
+state={
+    firstName:"",
+    lastName:"",
+    email:"",
+    phoneNumber:"",
+    password:"",
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: password should not be part of the form - they are not signing up for a secure account, they're just expressing their interest to volunteer.

File: client/src/components/VolunteersForm.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    address:""
+
+};
+onSubmit=e=>{
+    e.preventDefault();
+    console.log(this.state);
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: It's great that you added this until we can think of how we're going to send emails. It keeps this pull request small and easier to review! 👍

File: client/src/components/VolunteersForm.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            <br/>
+            <input placeholder="Phone number" 
+            value={this.state.phoneNumber}
+           onChange={e=> this.setState({phoneNumber: e.target.value})}
+            />
+            <br/>
+            <input placeholder="Password" 
+            value={this.state.password}
+           onChange={e=> this.setState({password: e.target.value})}
+            />
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: Again, I think the password field can be removed

File: client/src/containers/App/App.js [hunk 4]
```diff
@@ -31,6 +33,9 @@ const App = () => (
         <li>
           <Link to="/users">Users</Link>
         </li>
+        <li>
+          <Link to="/VolunteersForm">Volunteers form</Link>
```
Comment (rarmatei) [inline client/src/containers/App/App.js]: I would change this to `<Link to="/volunteers">Volunteer</Link>`

... to be more consistent with the other links in the header

File: client/src/containers/App/App.js [hunk 5]
```diff
@@ -40,6 +45,7 @@ const App = () => (
       <Route path="/shelters" component={Shelters} />
       <Route path="/referrals" component={Referrals} />
       <Route path="/users" component={Users} />
+      <Route path="/VolunteersForm" component={VolunteersForm} />
```
Comment (rarmatei) [inline client/src/containers/App/App.js]: ..if you change the above, you'll also need to change this to `<Route path="/volunteers" component={VolunteersForm} />`

File: client/src/components/VolunteersForm.js (hunk 1 above)
Comment (azadi11) [reply client/src/components/VolunteersForm.js]: yes you are right
thanks

File: client/src/components/VolunteersForm.js (hunk 2 above)
Comment (azadi11) [reply client/src/components/VolunteersForm.js]: ok .I do my all

File: client/src/containers/App/App.js (hunk 4 above)
Comment (azadi11) [reply client/src/containers/App/App.js]: yes that would be better
thanks

File: client/src/components/VolunteersForm.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+class VolunteersForm extends React.Component {
+state={
+    firstName:"",
+    lastName:"",
+    email:"",
+    phoneNumber:"",
+    yourExperience:"",
+    address:"",
+    yourExperience:""
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: the "yourExperience" field here has been added twice

File: client/src/components/VolunteersForm.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+
+class VolunteersForm extends React.Component {
+state={
+    firstName:"",
+    lastName:"",
+    email:"",
+    phoneNumber:"",
+    yourExperience:"",
+    address:"",
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: I don't think we need their address - could we please remove it?

File: client/src/components/VolunteersForm.js [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+            />
+            <br/>
+            <input placeholder="Phone number" 
+            value={this.state.phoneNumber}
+           onChange={e=> this.setState({phoneNumber: e.target.value})}
+            />
+            <br/>
+            <input placeholder="address" 
+            value={this.state.password}
+           onChange={e=> this.setState({password: e.target.value})}
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: if you remove address from the state, you'll need to remove this one as well

File: client/src/components/VolunteersForm.js [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+            />
+            <br/>
+
+            <textarea rows="4" cols= "22"
+            placeholder="your experience"
+            value={this.state.subject}
+           onChange={e=> this.setState({address: e.target.value})}
+           />
```
Comment (rarmatei) [inline client/src/components/VolunteersForm.js]: I think the above `<textarea>` is linked to wrong pieces of state - it should be:

```html
   <textarea rows="4" cols= "22"
            placeholder="your experience"
            value={this.state.yourExperience}
           onChange={e=> this.setState({yourExperience: e.target.value})}
           />
```

**Pull request merged**